src/sentry/releasehealth/sessions.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from typing import Optional, Sequence
+
+from sentry.releasehealth.base import ReleaseHealthBackend
+from sentry.snuba.sessions import get_current_and_previous_crash_free_rates
+
+
+class SessionsReleaseHealthBackend(ReleaseHealthBackend):
+    """Gets release health results from the session dataset"""
+
+    def get_current_and_previous_crash_free_rates(
+        self,
+        project_ids: Sequence[int],
+        current_start: datetime,
+        current_end: datetime,
+        previous_start: datetime,
+        previous_end: datetime,
+        rollup: int,
+        org_id: Optional[int] = None,
+    ) -> ReleaseHealthBackend.CurrentAndPreviousCrashFreeRates:
+        return get_current_and_previous_crash_free_rates(  # type: ignore
```

Comment:
Defining a type for `get_current_and_previous_crash_free_rates` will prevent us from needing to add `# type: ignore`